Source/DafnyRuntime/DafnyRuntimeGo-gomod/dafny/dafny.go

```diff
@@ -65,6 +66,35 @@ func AreEqual(x, y interface{}) bool {
 	}
 }
 
+// Checking for `Comparable` doesn't work, because it matches things for which == fails.
+func IsEqualityComparable(i interface{}) bool {
+	switch i.(type) {
+	case uint8, Char, CodePoint:
```

Comment:
Can you add a quick comment that this isn't an exhaustive list and we may want to add to it in the future?